packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1208,6 +1211,26 @@ export default class Meeting extends StatelessWebexPlugin {
       );
       this.reconnect();
     });
+
+    // There is two stats for mute one is the actual media being sent or received
+    // The second on is if the audio is muted, we need to tell the statsAnalyzer when
+    // the audio is muted or the user is not willing to send media
+    this.locusInfo.on(LOCUSINFO.EVENTS.MEDIA_STATUS_CHANGE, (status) => {
+      if (this.statsAnalyzer) {
+        this.statsAnalyzer.updateMediaStatus({
```

Comment:
```suggestion
        this.statsAnalyzer?.updateMediaStatus({
```

Reply:
same error , We cannot call that over a function